pkg/heatmap/axes/label.go

```diff
@@ -108,7 +108,7 @@ func xLabels(yEnd image.Point, graphWidth int, stringLabels []string, cellWidth
 
 // paddedLabelLength calculates the length of the padded label and
 // the column index corresponding to the label.
-// For example, the longest label's length is 5, like '12:34', and the cell's width is 3.
+// For examples, the longest label's length is 5, like '12:34', and the cell's width is 3.
```

Comment:
I don't think `for examples` is correct.
![image](https://user-images.githubusercontent.com/26627380/102612188-3fa4c080-416b-11eb-8359-d2c8c14f987f.png)
![image](https://user-images.githubusercontent.com/26627380/102612255-59460800-416b-11eb-9afe-9d7c71ab4dce.png)

Reply:
Neither do I, the IDE changed too many